src/scss/_variables.scss

```diff
@@ -13,4 +13,12 @@ $o-card-spacing-unit: 10px;
 
 /// Size of image when left aligned
 /// @type {Number}
-$o-card-image-width: 100px;
+$o-card-image-width: 40%;
+
+/// Colour of links and tags on Opinion cards
+/// @type {String}
+$o-card-opinion-colour: #416e92; // This should be put into o-colors
+
+$o-card-headshot-colour: #d0e2e7; // This should be put into o-colors
```

Comment:
When will this be put in `o-colors` ? Is it after Design decides on the standardisation of the tints? I'm pondering that myself for a different module.